content/config/otel.md

```diff
@@ -0,0 +1,164 @@
+for emitting metrics about command invocations. This is disabled by default.
+You can configure the CLI to start emitting metrics to the endpoint that you
+specify. This allows you to capture information about your `docker` command
+invocations for more insight into your Docker usage.
+
+The destination of these metrics is completely opt-in, and you control the
+destination to where the data is being sent.
+
+## What is OTel?
+
+OTel stands for Open Telemetry. OTel is an open observability framework for
+creating and managing telemetry data, such as traces, metrics, and logs.
+OpenTelemetry is vendor- and tool-agnostic, meaning that it can be used with a
+broad variety of Observability backends.
+
+Support for OTel instrumentation in the Docker CLI means that the CLI can emit
+information about events that take place, using the protocols and conventions
+defined in the Open Telemetry specification.
+
+## How it works
+
+The Docker CLI doesn't emit telemetry data by default. Only if you've set an
+environment variable on your system will Docker CLI attempt to emit OTel
+metrics, to the endpoint that you specify.
+
+```bash
+DOCKER_CLI_OTEL_EXPORTER_OTLP_ENDPOINT=<endpoint>
+```
+
+The variable specifies the endpoint of an OTel collector, where telemetry data
+about `docker` CLI invocation should be sent. To capture the data, you'll need
+an OTel collector listening on that endpoint.
+
+The purpose of a collector is to receive the telemetry data, process it, and
+exports it to a backend. The backend is where the telemetry data gets stored.
+You can choose from a number of different backends, such as Prometheus or
+InfluxDB.
+
+Some backends provide tools for visualizing the metrics directly.
+Alternatively, you can also run a dedicated frontend with support for
+generating more useful graphs, such as Grafana.
+
+## Setup
+
+To get started capturing telemetry data for the Docker CLI, you'll need to:
+
+- Set the `OTEL_EXPORTER_OTLP_ENDPOINT` environment variable to point to an OTel collector endpoint
+- Run an OTel collector that receives the signals from CLI command invocations
+- Run a backend for storing the data received from the collector
+
+The following Docker Compose file bootstraps a service to get started with
+OTel. It includes an OTel collector and a Prometheus backend.
+
+```yaml {collapse=true,title=compose.yml}
+name: cli-otel
+services:
+  prometheus:
+    image: prom/prometheus
+    command:
+      - "--config.file=/etc/prometheus/prom.yml"
+    ports:
+      # Publish the Prometheus frontend on localhost:9091
+      - 9091:9090
+    restart: always
+    volumes:
+      # Store Prometheus data in a volume:
+      - prom_data:/prometheus
+      # Mount the prom.yml config file
+      - ./prom.yml:/etc/prometheus/prom.yml
+  otelcol:
+    image: otel/opentelemetry-collector
+    restart: always
+    depends_on:
+      - prometheus
+    ports:
+      - 4317:4317
+      - 4322:4318
```

Comment:
is this port mapping necessary for the example?

Reply:
I don't think so... I will remove it. @laurazard is that right?

Reply:
Looks good! I must've left the other one there by mistake.